cmd/validator/slashing-protection/export.go

```diff
@@ -1,4 +1,4 @@
-package slashingprotection
+package slashingprotectioncmd
```

Comment:
Why the change? Other packages in `cmd` don't have the `cmd` suffix.

Reply:
The reason is that it avoids conflicts with the `slashingprotection` package when importing it. For example:
```go
import (
  // CONFUSION! Go imports autocomplete doesn't know what to do, need manual aliasing
  slashingprotection "github.com/prysm/cmd/validator/slashing-protection"
  slashingprotection "github.com/prysm/validator/slashing-protection"
)
```
vs
```go
import (
  // Automatically aliased by IDE
  slashingprotectioncmd "github.com/prysm/cmd/validator/slashing-protection"
  slashingprotection "github.com/prysm/validator/slashing-protection"
)
```